site/blog/starlingx-meetup-september-2019.md

```diff
@@ -0,0 +1,48 @@
+---
+title: News from the StarlingX Deep Dive Meetup in Beijing
+author: hai-tao-wang-yong-hu
+date: 2019-10-21T01:32:05.627Z
+category: news
+---
+
+Learn more about the recent StarlingX meetup in China. <!-- more -->
+
+The community had the first StarlingX meetup held in China in Beijing on September 20, 2019.
+
+More than 80 participants from 22 companies enrolled in this gathering with representatives from all major telecom operators, Cloud Service Providers and edge computing solution providers from the region. Experts from Intel, China Unicom, FiberHome, Wind River, 99Cloud, China UnionPay and China Mobile shared their use case of edge computing, and their experiences of the deploying and testing the software the community has been working on.
+
+And there is more! 4 demo showcases were performed to highlight different use cases, while the participants also engaged in an enthusiastic group discussion that turned the meetup into a collaborative environment to share current demands and challenges in the area of edge computing.
+
+## Case Studies
+
+In the morning, the meetup began with a joint presentation by four organizations (WindRiver, Intel, Fiberhome and 99Cloud), who are active contributors to StarlingX. This overall introduction painted a full picture from multiple viewpoints, from project history to community ecosystem nowadays, from use case scenarios to technical evolution on architecture, from the current 2.0 feature highlights to the outlook into the next and following releases.
+
+After the opening further presenters shared their tech solutions or product practices with StarlingX on the infrastructure layer.
+
+### Presentations
+
+**“StarlingX Practice in Industrial Manufacturing Edge Cloud”:**
+Besides being a traditional telecom operator, China Unicom is also pursuing CSP business opportunities on certain market segments, like industrial manufacturing on the edge. These environments demand platforms to connect the industrial fields with the cloud data centers with using edge and IOT technologies. The presentation outlined how IaaS, PaaS and SaaS solutions can provide a layered architecture to fulfill the needs of this use case.
+
+StarlingX is one of the favorable solutions on the IaaS layer, benefited from its dedicated optimizations for edge, such as, low latency, real-time, security and high availability, in addition to the orchestration capabilities inherit from Kubernetes and OpenStack. The speaker also highlighted an example how the Distributed Cloud feature can be used to deploy a cross-regional platform to fulfill the needs of the industrial manufacturing use case.
```

Comment:
StarlingX is one of the favorable solutions on the IaaS layer, benefiting from its dedicated optimizations for edge, such as, low latency, real-time, security and high availability, in addition to the orchestration capabilities inherited from Kubernetes and OpenStack. The speaker also highlighted an example how the Distributed Cloud feature can be used to deploy a cross-regional platform to fulfill the needs of the industrial manufacturing use case.